memgpt/memory.py

```diff
@@ -266,15 +269,17 @@ def search(self, query_string, count=None, start=None):
             return matches, len(matches)
 
     async def a_search(self, query_string, count=None, start=None):
+        # FIXME: The linter says that these parameters are wrong?
         return self.search(query_string, count=None, start=None)
 
 
 class DummyArchivalMemoryWithEmbeddings(DummyArchivalMemory):
     """Same as dummy in-memory archival memory, but with bare-bones embedding support"""
 
-    def __init__(self, archival_memory_database=None, embedding_model="text-embedding-ada-002"):
+    def __init__(self, archival_memory_database=None, embedding_model_arg="text-embedding-ada-002"):

```

Comment:
This is to avoid shadowing (I  know it is not important, but remember: Zero warnings policies catch bugs)

Reply:
I'm not sure why adding the `_arg` is important? Also FYI all the archival memory classes except for `EmbeddingArchivalMemory` will be deprecated / removed soon.

Reply:
Making it another name shows that you did not accidentally use the wrong name, because that name shadows the already existing variable in the context that has the same name. Doing it like that will avoid running in error when you refactor or actually do it by accident. So having such a rule in the linter saves your butt and as mentioned before: Zero warnings policies may safe the world one day :)